src/cli_plugin/install/pack.js

```diff
@@ -70,6 +69,10 @@ async function mergePackageData(settings, packages) {
     pkg.version = _.get(packageInfo, 'version');
     pkg.name = _.get(packageInfo, 'name');
     pkg.path =  resolve(settings.pluginDir, pkg.name);
+    pkg.kibanaVersion = _.get(packageInfo, 'kibana.version');
```

Comment:
I think this is a good place to add a comment explaining the intent behind this PR:

```javascript
// Plugins must specify the exact version of Kibana that they work with, but they also may have their own
// versioning scheme.
```